src/components/Counter.js

```diff
@@ -6,11 +6,14 @@ class Counter extends Component {
     incrementIfOdd = () => {
         // Stretch Problem: Implement an increment function that
         // only increments if the counter value is odd
+
+        this.props.count % 2 !== 0 ? this.props.increment() : console.log('the count is even')
```

Comment:
An interesting take on checking for truthfulness and ternary operator added in too.